evap/student/views.py

```diff
@@ -196,15 +238,19 @@ def get_vote_page_form_groups(request, evaluation, preview):
         questionnaires = contribution.questionnaires.all()
         if not questionnaires.exists():
             continue
-        form_groups[contribution] = [
-            QuestionnaireVotingForm(request.POST or None, contribution=contribution, questionnaire=questionnaire)
-            for questionnaire in questionnaires
-        ]
+        form_groups[contribution] = create_voting_forms(request, contribution, questionnaires, preselect_no_answer)
+
     return form_groups
 
 
-def render_vote_page(request, evaluation, preview, for_rendering_in_modal=False):
-    form_groups = get_vote_page_form_groups(request, evaluation, preview)
+def render_vote_page(
+    request: HttpRequest,
+    evaluation: Evaluation,
+    preview: bool,
+    dropout: bool,
+    for_rendering_in_modal: bool = False,
```

Comment:
As they are three bools now, let's make the keyword only (and also the other ones taking multiple preview and dropout bools):
```suggestion
    *,
    preview: bool,
    dropout: bool,
    for_rendering_in_modal: bool = False,
```